### PR TITLE
Pr/report

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/tocco-util/src/actions/utils/report.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types'
+
+const GROUP_GENERAL = 'generalSettings'
+const GROUP_RECIPIENT = 'recipientSettings'
+
+export const getGroupedValues = (settingsDefinition, values) => {
+  const groupValues = name => {
+    return {
```
Comment (rzueger) [inline packages/tocco-util/src/actions/utils/report.js]: superfluous return. line 10 again.

but that's fine with me - I guess readability is better this way?

File: packages/tocco-util/src/actions/utils/report.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        'id': 'recipientSettings',
+        'label': msg('client.common.report.recipientSettings'),
+        'layoutType': 'vertical-box',
+        children: extractFields(GROUP_RECIPIENT)
+      }
+    ]
+  }
+}
+
+export const reportSettingsDefinitionPropType = PropTypes.shape({
```
Comment (rzueger) [inline packages/tocco-util/src/actions/utils/report.js]: generalSettings, recipient or description required?

Comment (MethodenMann) [reply packages/tocco-util/src/actions/utils/report.js]: isn't it your definition that they can be null?

Comment (rzueger) [reply packages/tocco-util/src/actions/utils/report.js]: (Well, generalSettings will never be null probably.)
I just wanted to make sure that you're aware that they're not marked as required and that you're code is prepared to handle null values. Seems like that's the case. 👍

File: packages/tocco-util/src/actions/modules/actionHandlers/report.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        settingsDefinition={settingsDefinition}
+      />
+    ),
+    true
+  ))
+
+  const response = yield take(answerChannel)
+
+  // eslint-disable-next-line no-console
+  console.log('>>>>>response', response, entity, ids)
```
Comment (rzueger) [inline packages/tocco-util/src/actions/modules/actionHandlers/report.js]: can be removed?

Comment (MethodenMann) [reply packages/tocco-util/src/actions/modules/actionHandlers/report.js]: you are right, not really good to push this into master. I will implement report handling in this spring and merge it into this branch. Then we can merge everything together into master.
Will close this PR for now. thanks for reviewing

**Pull request closed without merging**